doc/nrf/protocols/thread/configuring.rst

```diff
@@ -144,16 +144,14 @@ At the end of the configuration process, you can check the EUI-64 value using Op
 
 .. _ug_thread_configuring_crypto:
 
-Hardware-accelerated cryptography options
-=========================================
+Cryptography options
+====================
 
-You can enable hardware-accelerated cryptography by using the :ref:`nrf_security` (nrf_security).
-To do this, modify the setting of the following Kconfig option:
+By default, the OpenThread stack uses the :ref:`nrf_security` (nrf_security) for cryptographic operations.
+To use `Mbed TLS`_, modify  the :kconfig:option:`OPENTHREAD_MBEDTLS_CHOICE` Kconfig option.
```

Comment:
I think the original doc entry was trying to inform on how to enable hardware acceleration (we need to use Nordic Security Module for that), which we're now missing. Also, I believe this config has more to do with mbedTLS configuration, rather than whether we use mbedTLS it or not. 
Previous version of doc is not up to date here, as configs have changed: Nordic Security Module is now selected by default. CONFIG_OPENTHREAD_MBEDTLS is now promptless and is now selectable only by OPENTHREAD_MBEDTLS_CHOICE config - this activates OpenThread's predefined mbedTLS configuration. I tried to build this config, but selecting it raises some errors, so I'm not sure it's a recommended path.
I think we should either just state that default configuration uses Nordic Security Module (capable of hardware accelerated crypto operations) or rewrite this section to provide some more information about Thread cryptography configurations in NCS (nRF Security - default / Custom / mbedTLS built-in).